nva-datamodel-java/src/main/java/no/unit/nva/model/time/duration/DefinedDuration.java

```diff
@@ -0,0 +1,105 @@
+package no.unit.nva.model.time.duration;
```

Comment:
There is an ISO standard for duration too.

So,
```
"duration": {
  "type": "DefinedDuration",
  "minutes": "35",
  "hours": "2",
  "days": "0",
  "weeks": "0"
}
```
could be written as:
```
"duration": "PT2H35M"
```
Anyway, if it is a well-defined duration, `minutes`, `hours`, `days` and `weeks` should be integers.